dashboard/src/components/TablePaginationSection.tsx

```diff
@@ -2,8 +2,41 @@ import KeyboardArrowLeftIcon from "@mui/icons-material/KeyboardArrowLeft";
 import KeyboardArrowRightIcon from "@mui/icons-material/KeyboardArrowRight";
 import { Box, Button, IconButton, iconButtonClasses } from "@mui/joy";
 
-export const PaginationSection = () => {
-  //TODO: Implement pagination logic. Try to use cached data from query hook using query key
+interface TablePaginationSectionProps {
+  currentPage: number;
+  pageCount: number;
+  onPageChange: (page: number) => void;
+}
+
+export const TablePaginationSection: React.FC<TablePaginationSectionProps> = ({
```

Comment:
spicy :hot_pepper:

Reply:
missed that React.FC part. Will change it to FC